spec/unit/room.spec.ts

```diff
@@ -168,30 +168,45 @@ describe("Room", function () {
             room.client,
         );
 
-    const addRoomMainAndThreadMessages = (
-        room: Room,
-        tsMain?: number,
-        tsThread?: number,
-    ): { mainEvent?: MatrixEvent; threadEvent?: MatrixEvent } => {
-        const result: { mainEvent?: MatrixEvent; threadEvent?: MatrixEvent } = {};
-        const { rootEvent, thread } = mkThread({
+    /**
+     * Starts a new thread in a room by creating a message as thread root.
+     * Also creates a Thread model and adds it to the room.
+     * Does not insert the messages into a timeline.
+     */
+    const mkThread = (args: Partial<typeof threadUtils.mkThread>) => {
```

Comment:
really?

`threadUtils.mkThread` is a function type, so this is saying `args` is of type `Partial<Function<...>>` ?

Reply:
Done